src/components/TextInputWithDropdown/TextInputWithDropdown.tsx

```diff
@@ -52,12 +60,14 @@ const TextInputWithDropdown = () => {
         excludeDigits
         onClick={handleCountryInputClick}
         inputRef={inputRef}
+        dataTestId={dataTestId}
```

Comment:
**nitpick:** No need to pass dataTest this deep, we can do it directly in component